contracts/src/exchanges/ekubo/launcher.cairo

```diff
@@ -379,5 +415,47 @@ mod EkuboLauncher {
             // Clear this contract and send the tokens back to the caller
             token.transfer(recipient: caller, amount: token.balanceOf(this));
         }
+
+        /// Ensures that the caller is the owner of the specified position.
+        ///
+        /// # Arguments
+        ///
+        /// * `position_to_transfer` - The LP position to transfer.
+        ///
+        /// # Panics
+        ///
+        /// This function will panic if:
+        ///
+        /// * The caller's address is not the same as the `owner` of the `position_to_transfer` (error code: `NOT_POSITION_OWNER`).
+        ///
+        fn assert_only_position_owner(self: @ContractState, position_to_transfer: StorableEkuboLP) {
+            let owner_of_position: ContractAddress = position_to_transfer.owner;
+            assert(get_caller_address() == owner_of_position, NOT_POSITION_OWNER);
+        }
+
+        /// Removes the id of a position from the list of positions of a user.
+        ///
+        /// Internally, this function reads the list of positions of the specified `owner` from the 'owner_to_positions' mapping.
+        /// It then iterates over the list and replaces the specified `position` with the last element of the list.
+        /// The length of the list is then decremented by one, and the last element of the list is set to zero.
+        fn remove_position_from_list(self: @ContractState, position: u64, mut list: List<u64>) {
+            let list_len = list.len();
+            let mut i = 0;
+            loop {
+                if i == list_len {
+                    break;
+                }
+                let current_position = list[i];
+                if current_position == position {
```

Comment:
usually we try to code with the least amount of logic inside if branches. so 

if current position == position {
  // do a lot of stuff
}

would become

```rust
if current_position != position{
    continue;
}
// do a lot of stuff

which makes it easier to process in case of intricate loops

Reply:
oh ok, i just copy pasted the locked helper function, will change that

Reply:
yeah no big deal just clean code conventions